src/Apps/Collect/Components/Seo/SeoProductsForArtworks.tsx

```diff
@@ -0,0 +1,148 @@
+import currency from "currency.js"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { toSentence } from "underscore.string"
+
+import { SeoProductsForArtworks_artworks } from "__generated__/SeoProductsForArtworks_artworks.graphql"
+import { Product } from "./Product"
+
+const { APP_URL } = process.env
+
+const AVAILABILITY = {
+  "for sale": "https://schema.org/InStock",
+  sold: "https://schema.org/OutOfStock",
+}
+
+const formatCurrency = value => currency(value, { separator: "" }).format()
+
+interface SeoProductsProps {
+  artworks: SeoProductsForArtworks_artworks
+}
+
+export class SeoProducts extends React.Component<SeoProductsProps> {
+  render() {
+    const {
+      artworks: { artworks_connection },
+    } = this.props
+
+    // here the filtering is necessary so we can re-use the artwork list shown in the page (could include
+    // non-acquireable artworks) without making an extra request.
+    const artworksForSeoProduct = artworks_connection.edges.filter(
+      edge => edge.node.is_acquireable && edge.node.partner
+    )
+
+    return artworksForSeoProduct.map(({ node }, index) => {
+      const { artists, is_price_range, partner, price } = node
+      const location = partner.locations && partner.locations[0]
```

Comment:
Here the location is picked up based on the array index, which may not be the location where the piece is stored, but I wasn't sure how I could pick the right location.

Reply:
Does the seller need to be limited to one?

Reply:
Good question. From our perspective, no, but structured data may not allow the data to have multiple sellers. I'll double-check once I have access to Google (I'm in Beijing and have no access to Google)